metabase/metabase
# fix visual issues on dashboard tabs

Part of epic https://github.com/metabase/metabase/issues/29502

### Description

This PR fixes a few small visual issues (font sizes, alignments, etc) for dashboard tabs. These are outlined in [this document](https://www.notion.so/metabase/Bring-tabbed-navigation-to-dashboards-dd2e71b4d1954f8188ecbd9914e81420#17219475a71c40529cf04ff41d1d4080) and [this thread](https://metaboat.slack.com/archives/C057T1QTB3L/p1684936928237019).

### How to verify

1. Create dashboard with tabs
2. Verify styles match [figma](https://www.figma.com/file/DuwiUCVo1K4EUBw50pGYPy/Tabs-in-dashboards?type=design&node-id=104-5378&t=MuhT58CrxPQCanOn-0)

### Demo

https://github.com/metabase/metabase/assets/37751258/e9cd39a1-87fc-4fee-bb22-a5f531a16f24

### Checklist


_Can't really test style changes and there are existing unit and e2e tests for this feature._
